submissions/KostiantynOkhorzin/memory-game/css/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+html, body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 16px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+input, button, textarea {
+  font: inherit;
+}
+
+button {
+  cursor: pointer;
+  background: inherit;
+}
+
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.game__title {
+  font-size: 35px;
+  font-weight: 900;
+  line-height: 160%;
+  text-align: center;
+}
+
+@media (max-width: 425px) {
+  .game__title {
+    font-size: 28px;
+  }
+}
+
+.game__body {
+  display: grid;
+  grid-template-columns: repeat(4, 150px);
+  grid-template-rows: repeat(3, 150px);
+  perspective: 1000px;
+}
+
+@media (max-width: 768px) {
+  .game__body {
+    grid-template-columns: repeat(4, 100px);
+    grid-template-rows: repeat(3, 100px);
+  }
+}
+
+@media (max-width: 425px) {
```

Comment:
You already have a breakpoint of `425px` on the line 56. Just place this code there, no need to create separate `@media` blocks for the same width.